ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      netRegex: NetRegexes.ability({ source: '4th Legion Augur', id: '5F1B', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Get Towers',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Augur Turbine',
+      netRegex: NetRegexes.startsUsing({ source: 'Flameborne Zirnitra', id: '5F14' }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Stack + Knockback to Safe Spot',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Alkonost Stormcall',
+      netRegex: NetRegexes.startsUsing({ source: 'Tamed Alkonost', id: '5F26', capture: false }),
+      delaySeconds: 2,
+      durationSeconds: 5,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: we could figure out where this orb is and say go north/south.
+          en: 'Go Opposite Fast Orb',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Alkonost Wind',
+      // 5F21 = North Wind
+      // 5F22 = South Wind
+      netRegex: NetRegexes.startsUsing({ source: 'Tamed Carrion Crow', id: ['5F21', '5F22'] }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Zadnor Alkonost Stormcall Away',
+      netRegex: NetRegexes.startsUsing({ source: 'Tamed Alkonost', id: '5F26', capture: false }),
+      delaySeconds: 18,
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Orb',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Alkonost Nihility\'s Song',
+      netRegex: NetRegexes.startsUsing({ source: 'Alkonost', id: '5F28', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Cuchulainn March',
+      // 871 = Forward March
+      // 872 = About Face
+      // 873 = Left Face
+      // 874 = Right Face
+      netRegex: NetRegexes.gainsEffect({ source: '4th-Make Cuchulainn', effectId: ['871', '872', '873', '874'] }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => {
+        const effectId = matches.effectId.toUpperCase();
+        if (effectId === '871')
+          return output.forward();
+        if (effectId === '872')
+          return output.backward();
+        if (effectId === '873')
+          return output.left();
+        if (effectId === '874')
+          return output.right();
+      },
+      outputStrings: {
+        forward: {
+          en: 'March Forward (avoid puddles)',
+        },
+        backward: {
+          en: 'March Backward (avoid puddles)',
+        },
+        left: {
+          en: 'March Left (avoid puddles)',
+        },
+        right: {
+          en: 'March Right (avoid puddles)',
+        },
```

Comment:
Should these just tell you your debuff? I think this is counter to how we usually implement these things.

I think the one earlier in the file is okay since you have to "aim" but this one sounds like there is "general floor bad" and you also get a march mechanic

Reply:
This is the only thing going on at this point that you have to do with a march.  If you do pick up a puddle, you're often turned into an extremely slow slime during the next mechanic which will likely kill you.  Agreed that it's long, but that's the reasoning.